CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ option(ENABLE_PLUGIN_CGROUP_NETWORK "enable cgroup-network plugin" ${DEFAULT_FEA
 option(ENABLE_PLUGIN_CUPS "enable cups.plugin" ${DEFAULT_FEATURE_STATE})
 option(ENABLE_PLUGIN_DEBUGFS "enable debugfs.plugin" ${DEFAULT_FEATURE_STATE})
 option(ENABLE_PLUGIN_EBPF "enable ebpf.plugin" ${DEFAULT_FEATURE_STATE})
+cmake_dependent_option(ENABLE_LEGACY_EBPF_PROGRAMS "Enable eBPF programs for older kernels without BTF support" True ENABLE_PLUGIN_EBPF False)
```

Comment:
Nowadays, it is also possible to have kernels without `BTF`, so I suggest removing the `older`.